.eslintrc.js

```diff
@@ -7,10 +7,25 @@ module.exports = {
   },
   plugins: [
     'eslint-plugin-tsdoc',
+    'eslint-plugin-import',
+    'eslint-plugin-unused-imports'
   ],
   extends: [
     'es/node',
+    'plugin:import/errors',
+    'plugin:import/warnings',
+    'plugin:import/typescript'
   ],
+  settings: {
+    'import/parsers': {
+      '@typescript-eslint/parser': ['.ts', '.tsx']
+    },
+    'import/resolver': {
+      'typescript': {
+        'alwaysTryTypes': true // always try to resolve types under `<root>@types` directory even it doesn't contain any source code, like `@types/rdf-js`
+      },
+    }
```

Comment:
Can you explain what this one does? The linter gives no errors if I remove it and I can't seem to find any extended documentation on it online.

Reply:
Copy-pasted from Comunica, where I added those line because the linter complained there was no `rdf-js` package. Maybe the problem only occurs in some cases, which we do not (yet) have in the server.